pkg/prometheus/prom-query.go

```diff
@@ -56,19 +58,24 @@ func (pq *promQuery) execute(jobCtxt context.Context) (proto.Message, error) {
 		ctx, cancel := context.WithTimeout(jobCtxt, pq.timeout)
 		defer cancel()
 
-		result, warnings, err = pq.promAPI.Query(ctx, pq.query, pq.endTimestamp)
+		query, innerErr := pq.enforcer.EnforceLabels(pq.query)
+		if innerErr != nil {
```

Comment:
why not simply 'err'?

Reply:
It's shadowin err from outer scope, and golint-ci is complaining